### DISCUSSION TITLE
Guidance on how to allow WebView to participate in RNGH's Gesture System

### DISCUSSION THREAD

**Discussion by alexchen5** [software-mansion/react-native-gesture-handler#2277] [Q&A]
I want touch events inside WebView to also be registered by my app, but I've found a lack of documentation on how to do so. 
Currently I have a component structured like so: 

```
import { Gesture, GestureDetector } from "react-native-gesture-handler"
import { WebView } from "react-native-webview"

<GestureDetector 
    gesture={Gesture.Tap().onStart(() => console.log("tapped!"))}
>
    <WebView {...props} />
</GestureDetector>
```

I'm unable to get my `GestureDetector` to detect any gestures from within `WebView`. 

I have a feeling the solution has something to do with using `NativeGesture`, however the docs don't really say much and I can't find any examples of the API being used anywhere else. 

Some guidance on how I should compose the gesture would be really helpful <3

**Comment by alexchen5** [comment]
I found a workaround to add `pointerEvents="none"` to the WebView props... which works well but I'd still be keen to know what the `NativeGesture` solution is if anyone wants to help.

**Reply by AlexSapoznikov** [reply]
This doesn't work for me

**Comment by delmaass** [comment]
Wrap your `WebView` with a `Native` gesture (will "connect" to the gesture events on your webview), then allow simulataneous gestures on your top gesture.

```
export default function Index() {
  const webViewGesture = Gesture.Native();

  const gesture = Gesture.Pan()
    .simultaneousWithExternalGesture(webViewGesture)
    .onUpdate((event) => {
      console.log(event.translationY);
    });

  return (
    <GestureDetector gesture={gesture}>
        <GestureDetector gesture={webViewGesture}>
          <WebView ...  />
        </GestureDetector>
    </GestureDetector>
  );
}
```
